leapt/im-bundle/2.0/config/packages/leapt_im.yaml

```diff
@@ -0,0 +1,6 @@
+leapt_im:
+    cache_path: cache/im
+    # You can define reusable formats here
+    formats:
+        #my_format:
+        #    thumbnail: 350x350^
```

Comment:
Could we add something useful here instead of the comments?

Like: 
```
  formats:
    # resizes to fit in 100x100, only if bigger, and remove all crap (meta-data, ...)
    medium:
      thumbnail: 100x100>
```

Reply:
Nice idea :)